### ISSUE TITLE
Handle out-of band EntropyRequest

### ISSUE THREAD

**Issue by olemis** [skycoin/hardware-wallet-go#127]
**Describe the bug**
Handle out-of-band entropy requests.

**Environment information:**

- OS: all
- Platform: discovered on `Linux name 4.19.0-1-amd64 1 SMP Debian 4.19.12-1 (2018-12-22) x86_64 GNU/Linux`
- Firmware Version: skycoin/hardware-wallet@ffd6fcd 

**Steps to Reproduce**

Run integration tests against physical Skywallet device .

**Actual behavior**

This is the output of a `go run cmd/cli/cli.go features` for example:

```
[2019-05-10T21:28:35-04:00] INFO [walletusb]: bLength 18
bDescriptorType 1
bcdUSB 2.00
bDeviceClass 0
bDeviceSubClass 0
bDeviceProtocol 0
bMaxPacketSize0 64
idVendor 0x313a
idProduct 0x0001
bcdDevice 1.00
iManufacturer 1
iProduct 2
iSerialNumber 3
bNumConfigurations 1
[2019-05-10T21:28:35-04:00] ERROR [walletusb]: current configuration 1
[2019-05-10T21:28:35-04:00] INFO [walletusb]: finishing read queue
[2019-05-10T21:28:35-04:00] ERROR [skycoin-hw-cli]: received unexpected message type: MessageType_EntropyRequest
```

**Expected behavior**
No matter what command is executing , the entropy message should be either ignored (not recommended) or handled by sending `EntropyAck` back to the physical device.

**Additional context**
Initially reported in https://github.com/skycoin/hardware-wallet-go/pull/126#issuecomment-491469255

**Possible implementation**
beware of the fact that this affects all possible interactions with the device , so maybe 

- [ ] Handle `EntropyRequest` gracefully by ignoring it
- [ ] If handling it, add a separate goroutine to receive `EntropyRequest` and reply with `EntropyAck`

**Comment by therealssj** [comment]
im not able to reproduce this
can you tell list the steps for building the firmware to triggere entropy request?

**Comment by olemis** [comment]
looks like fixed in 5e94c59
